### DISCUSSION TITLE
⛔ Animation improvements

### DISCUSSION THREAD

**Discussion by no-name-99** [elementor/elementor#24113] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It seems that, Elementor is same what limited on animation …. Improving the animation, adding more in-depth settings for this is bring a new level of creativity and customisation 

### Describe the solution you'd like

I do not want to promote other web building solutions but, it seems that Framer and Webflow do have a more advanced animation for the elements within the websites

### Describe alternatives you've considered

Framer or Webflow

**Comment by nicholaszein** [comment]
Hi @no-name-99!

Thank you for taking the time so share your feedback! It means a lot to us! 🙏 

While we appreciate your enthusiasm, we find your suggestions too broad, and difficult to track. For this reason we are closing this discussion in favor of other similar discussions that are more specific.

Please, check the list of Feature Requests related to Animations in [this link](https://github.com/orgs/elementor/discussions/categories/feature-request?discussions_q=is%3Aopen+category%3A%22Feature+Request%22+animations), and vote for the ones you wish to be added to Elementor. The more votes, the better the chances we are going to implement them!

Thank you for your understanding! 🙏
